Hw_05/hw05.md

```diff
@@ -0,0 +1,155 @@
+January 20th 2020. Though the trend line for unscented candles does
+begin to trend down after January 20th, it does not appear to deviate
+enormously from what would be expected results. The average ratings for
+scented candles tell a different story. The trend line experiences a
+large drop off after January 20th 2020. This large decrease in average
+ratings does not appear to be normal for the data and could be related
+to individuals not realizing they have lost their sense of smell due to
+contracting COVID-19.
+
+## Part 2: Combined plot
+
+``` r
+candles_full <- bind_rows(
+    scented = Scented_all, 
+    unscented = Unscented_all, 
+    .id = "Scented"
+)
+```
+
+``` r
+Combined_Plot <- candles_full %>%
+  filter(Date > "2017-01-01") %>%
+  group_by(Date, Scented) %>% 
+  mutate(Avg_Rating = mean(Rating)) %>% 
+  ggplot(
+    aes(x = as.Date(Date),
+        y = Avg_Rating,
+        shape = Scented,
+        color = Scented,
+        fill = Scented
+  )) + 
+  geom_point(alpha = .3) +
+  scale_x_date(name = "Date", date_breaks = "6 months", date_labels = "%m-%Y") +
+  geom_smooth(method = "loess") +
+  geom_vline(xintercept = as.numeric(as.Date("2020-01-20")), linetype = "dashed") +
+  scale_y_continuous(name = "Average Rating") +
+  theme_minimal() +
+  ggtitle("Average Candle Reviews For Scented and Unscented Candles") + 
+  guides(x = guide_axis(n.dodge = 2)) + 
+  scale_color_viridis_d()
+
+Combined_Plot
+```
+
+    ## `geom_smooth()` using formula 'y ~ x'
+
+![](hw05_files/figure-gfm/unnamed-chunk-2-1.png)<!-- -->
+
+The combined plot is easier to interpret specifically if you are trying
+to compare the reviews of unscented and scented candles. Having the data
+overlap and specifically having the trend lines on the same graph allows
+for a very clear comparison of the two average ratings. Otherwise
+individuals would have to compare across the graphs and with both trend
+lines trending towards the top of the graph it may be harder to make
+accurate comparisons.
+
+## Part 3: Proportion of reviews mentioning “no scent”
+
+``` r
+Scented2_Plot <- Scented_all %>%
+  filter(Date >= "2019-01-01") %>%
+  mutate(month = lubridate::floor_date(Date, "month"),
+         noscent = str_no_scent(Review),
+         year = ifelse(Date <= as.Date("2019-12-31"),"2019", "2020")) %>% 
+  group_by(as.Date(month), noscent) %>% 
+  ggplot() +  aes(x = as.Date(month), y = (..count..)/sum(..count..), color = as.factor(year), fill = as.factor(year)) +
+  geom_bar(alpha = .4) +
+  stat_summary(aes(y = noscent), fun.data = mean_cl_normal) +
+  scale_x_date(name = "Month and Year", date_labels = "%m-%Y", date_breaks = "month", expand = c(0,0)) + 
+  guides(x = guide_axis(n.dodge = 3), color = guide_legend(title = "Year"), fill = guide_legend(title = "Year")) +
+  scale_y_continuous(name = "Percentage of Reviews mentioning 'No Scent'") +
+  labs(title = "Percentage of Scented Candle Reviews Mentioning 'No Scent' in 2019 and 2020")
+
+
+Scented2_Plot
```

Comment:
For some reason the plot shown here is from an earlier attempt and does not match the plot that I got when running this code written above.